bazel_tools/java.bzl

```
@@ -29,10 +30,35 @@ java_home_runtime = repository_rule(
 def _wrap_rule(rule, name = "", **kwargs):
     rule(name = name, **kwargs)
 
-def da_java_library(name, **kwargs):
-    _wrap_rule(native.java_library, name, **kwargs)
+def da_java_library(name, deps, srcs, data = [], resources = [], resource_jars = [], resource_strip_prefix = None,
+                    tags = [], visibility = None, exports = []):
+
+    root_packages = None
+    for tag in tags:
+       if tag.startswith("javadoc_root_packages="):
+            root_packages = [ tag[len("javadoc_root_packages="):] ]
```

```suggestion
            root_packages.append(tag[len("javadoc_root_packages="):])
```
Did you mean to append to this list?

---

I've now changed this to split on the tag element to generate the list.